builder/cloudstack/step_create_instance.go

```diff
@@ -65,7 +115,29 @@ func (s *stepCreateInstance) Run(state multistep.StateBag) multistep.StepAction
 	}
 
 	if config.UserData != "" {
-		ud, err := getUserData(config.UserData, config.HTTPGetOnly)
+		var httpPort uint = state.Get("http_port").(uint)
+		hostIp, err := hostIP()
+		if err != nil {
+			state.Put("error", err)
```

Comment:
I see. I don't know, but I saw that all of the hyperv, paralels, qemu, virtualbox and vmware providers do this right after setting ```common.SetHTTPIP(hostIp)```. I can remove it, and I don't know what we'll be missing out on. :-) @svanharmelen